mailpoet/assets/js/src/email-editor/engine/components/notices/sent-email-notice.tsx

```diff
@@ -0,0 +1,31 @@
+import { dispatch, useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import { store as noticesStore } from '@wordpress/notices';
+import { __ } from '@wordpress/i18n';
+
+import { storeName } from '../../store';
+
+export function SentEmailNotice() {
+  const { isEmailSent } = useSelect(
+    (select) => ({
+      isEmailSent: select(storeName).isEmailSent(),
+    }),
+    [],
+  );
+
+  useEffect(() => {
+    if (isEmailSent) {
+      dispatch(noticesStore).createErrorNotice(
```

Comment:
Have you considered the use of a warning instead of an error?
The warning notice seems better to me because it looks better and is more accurate to this type of message.

Reply:
Hmm, the older email editor displays it as an error, so I made it the same: https://d.pr/i/yrQ6KB

warning is also ok: https://d.pr/i/MuLO7y

I will change it.